embulk-cli/src/main/sh/selfrun.sh

```diff
@@ -89,5 +89,5 @@ else
     java_args="-XX:+AggressiveOpts -XX:+TieredCompilation -XX:TieredStopAtLevel=1 -Xverify:none $java_args"
 fi
 
-exec java $java_args -jar "$0" $jruby_args "$embulk_args"
+exec java $java_args -jar "$0" $jruby_args $embulk_args
```

Comment:
minor: Does `"$0"` still need `"` quote character? If we could remove `"` from `"$embulk_args"`, I thought that we can remove `"` from `"$0"`.